Shared/Extensions/JellyfinAPI/DeviceType.swift

```diff
@@ -0,0 +1,176 @@
+            self = .homeAssistant
+        case "Jellyfin Roku":
+            self = .roku
+        case "Finamp":
+            self = .finamp
+        case "Jellyfin Web", "Jellyfin Web (Vue)":
+            self = DeviceType(webBrowser: deviceName)
+        default:
+            self = .other
+        }
+    }
+
+    // MARK: - Initialize the Browser if Jellyfin-Web
+
+    private init(webBrowser: String?) {
+        switch webBrowser {
+        case "Opera", "Opera TV", "Opera Android":
+            self = .opera
+        case "Chrome", "Chrome Android":
+            self = .chrome
+        case "Firefox", "Firefox Android":
+            self = .firefox
+        case "Safari", "Safari iPad", "Safari iPhone":
+            self = .safari
+        case "Edge Chromium", "Edge Chromium Android", "Edge Chromium iPad", "Edge Chromium iPhone":
+            self = .edgechromium
+        case "Edge":
+            self = .edge
+        case "Internet Explorer":
+            self = .msie
+        default:
+            self = .html5
+        }
+    }
+
+    // MARK: - Client Image
+
+    var systemImage: String {
+        rawValue
+    }
+
+    // MARK: - Client Color
+
+    var clientColor: Color {
+        switch self {
+        case .samsungtv:
+            return Color(red: 0.0, green: 0.44, blue: 0.74) // Samsung Blue
+        case .xbox:
+            return Color(red: 0.0, green: 0.5, blue: 0.0) // Xbox Green
+        case .playstation:
+            return Color(red: 0.0, green: 0.32, blue: 0.65) // PlayStation Blue
+        case .kodi:
+            return Color(red: 0.0, green: 0.58, blue: 0.83) // Kodi Blue
+        case .android:
+            return Color(red: 0.18, green: 0.8, blue: 0.44) // Android Green
+        case .apple:
+            return Color(red: 0.35, green: 0.35, blue: 0.35) // Apple Gray
+        case .homeAssistant:
+            return Color(red: 0.0, green: 0.55, blue: 0.87) // Home Assistant Blue
+        case .roku:
+            return Color(red: 0.31, green: 0.09, blue: 0.55) // Roku Purple
+        case .finamp:
+            return Color(red: 0.61, green: 0.32, blue: 0.88) // Finamp Purple
+        case .chrome:
+            return Color(red: 0.98, green: 0.75, blue: 0.18) // Chrome Yellow
+        case .firefox:
+            return Color(red: 1.0, green: 0.33, blue: 0.0) // Firefox Orange
+        case .safari:
+            return Color(red: 0.0, green: 0.48, blue: 1.0) // Safari Blue
+        case .edgechromium:
+            return Color(red: 0.0, green: 0.45, blue: 0.75) // Edge Chromium Blue
+        case .edge:
+            return Color(red: 0.19, green: 0.31, blue: 0.51) // Edge Gray
+        case .msie:
+            return Color(red: 0.0, green: 0.53, blue: 1.0) // Internet Explorer Blue
+        case .opera:
+            return Color(red: 1.0, green: 0.0, blue: 0.0) // Opera Red
+        default:
+            return Color.black
+        }
+    }
+
+    // MARK: - Client Title
+
+    var displayTitle: String {
```

Comment:
I disagree that these should be localized as these are proper nouns. I don't have the time to see if every single brand here has opted into localized branding, but at least Android, Apple, and Firefox don't.